tcp_check/datadog_checks/tcp_check/tcp_check.py

```diff
@@ -81,16 +61,9 @@ def __init__(self, name, init_config, instances):
             'instance:{}'.format(self.instance_name),
         ]
 
-        # IPv6 address format: 2001:db8:85a3:8d3:1319:8a2e:370:7348
-        if is_ipv6(self.host):  # It may then be a IP V6 address, we check that
-            # It's a correct IP V6 address
-            self.socket_type = socket.AF_INET6
-        else:
-            self.socket_type = socket.AF_INET
-            # IP will be resolved at check time
-
     @property
     def addrs(self):
+        # type: (Any) -> List[namedtuple(str, socket.AddressFamily)]
```

Comment:
I think this should be `()` instead of `(Any)`, as it doesn't take arguments. Also, it's weird to use `namedtuple(...)` as a type. If we define the namedtuple outside as `AddrTuple`, then we can turn this into:

```suggestion
        # type: () -> List[AddrTuple]
```

Which is quite nicer.